src/components/InfinityTokensModal/InfinityTokensModal.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useEffect, useState } from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import PoolBuilderDie from '../PoolBuilder/PoolBuilderDie';
+import { DOOM, MOMENTUM } from '../../consts/diceConstants';
+import styles from './InfinityTokensModal.module.css';
+
+function InfinityTokensModal({
```

Comment:
It seems like a copy-paste of ConanTokensModal. If it's the same it would be better to use it as a common component instead of copying it here.

Reply:
The tokens are different so maybe it could be made a generic tokens module and then get which game it is passed in
Note that new tokens have now been made